proto/frequenz/api/common/v1/pagination/pagination_params.proto

```diff
@@ -11,16 +11,18 @@ package frequenz.api.common.v1.pagination;
 
 // A message defining parameters for paginating list requests.
 // It can be appended to a request message to specify the desired page of
-// results and the maximum number of results per page. For initial requests
-// (requesting the first page), the page_token should not be provided. For
-// subsequent requests (requesting any page after the first), the
-// next_page_token from the previous responses PaginationInfo must be supplied.
-// The page_size should only be specified in the initial request and will be
-// disregarded in subsequent requests.
+// results and the maximum number of results per page.
 message PaginationParams {
-  // The maximum number of results to be returned per request.
-  optional uint32 page_size = 1;
+  oneof params {
+    // The maximum number of results to return in a single page. The service may
+    // return fewer results than requested. If unspecified, the service will
```

Comment:
I would leave the default behaviour up to each service and not specify here that the service will choose a reasonable default.

Reply:
You mean, whether its optional or not should be up to the service?

Reply:
Yes